unstructured/partition/auto.py

```diff
@@ -193,6 +193,7 @@ def partition(
             file=file,  # type: ignore
             url=None,
             include_page_breaks=include_page_breaks,
+            infer_table_structure=pdf_infer_table_structure,
```

Comment:
Only thing that's a little weird here is that we're passing `pdf_infer_table_structure` to `partition_image`.

Reply:
yeah, i can see the case for wanting `image_infer_table_structure`, `pdf_infer_table_structure`, and `other_infer_table_structure` given that image table inference is still expensive (especially for many docs), but not as expensive as large PDF docs. but that is getting awkward.

one solution to slowly move to a new top level of `docs_types_no_infer_table_structure` that has `pdf` as the default, but could be `pdf,jpg,png,tiff`?

[one subtlety to note is that tiff's tend to be many page, more like PDF's]

Reply:
Yeah I'm game for that, and I like that specify a setting per document type. I'd probably go for a shorter kwarg name though, like `skip_infer_table`

Reply:
do i need to add `tiff`? i don't think we support it right now.

Reply:
Don't need to add `tiff` in this PR, I can create a follow on issue for that

Reply:
#977

Reply:
another question: in order to use table extraction for PDF, a user has to set `pdf_infer_table_structure` to true, and also set `skip_infer_table_types` to not include `"pdf"`, is that too much trouble for them to use?

Reply:
What about `pdf_infer_table_structure` take precedence for PDFs if it's set to `True`, and logging out a warning that says that you should use `skip_infer_table_types` to turn on table extraction for PDFs instead?